config/database.yml

```
@@ -1,7 +1,8 @@
 default: &default
   adapter: postgresql
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  timeout: 5000
+  variables:
+    statement_timeout: 10000
```

```suggestion
    statement_timeout: 10000
    idle_in_transaction_session_timeout: 10000
```

@vigneshmsft like this?

---

yup, looks good to me